pom.xml

```diff
@@ -124,4 +165,83 @@
     </plugins>
   </reporting>
 
+  <profiles>
+    <profile> 
+      <id>release</id>
```

Comment:
+0, the link actually adds noise to the configuration file because it talks about what profiles are and how they work--we have a link to the guide--a deployer should have that context.